### PR TITLE
Clean up .sln files

### PR DESCRIPTION
**Pull request by rynowak**
Remove reference to deleted DNX project
Prevent sources packages from building

### CODE REVIEW

**Comment by rynowak** [comment]
FYI only

File: Mvc.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+VisualStudioVersion = 14.0.25123.0
```
Comment (pranavkm) [inline Mvc.sln]: Could you write a test for this change?

**Comment by pranavkm** [comment]
:watch:

**Comment by javiercn** [comment]
LGTM :trollface:

**Pull request closed without merging**